docs/v1/P4Runtime-Spec.mdk

```diff
@@ -5168,7 +5167,7 @@ message SetForwardingPipelineConfigRequest {
     RECONCILE_AND_COMMIT = 5;
   }
   uint64 device_id = 1;
-  uint64 role_id = 2;
+  uint64 role = 2;
```

Comment:
Again doesn't compare to pb file, should be 'string role = 6' I think?

Reply:
Thanks for catching this!